tests/conftest.py

```diff
@@ -3622,3 +3622,10 @@ def teardown():
     es = ElasticSearch()
 
     return es
+
+
+def ceph_fs_use():
+    """
+    Bytes used value saved by cephfs small file scaling test.
+    """
+    return 0
```

Comment:
Is this function is called somewhere? didn't get the purpose of it.

Reply:
I got rid of this.